### PR TITLE
Patch release v.2.0.3

### PR DESCRIPTION
**Pull request by CarlosNZ**
Bump version for patch release

### CODE REVIEW

**Review by roxy-dao — APPROVED** [review]
Thanks!

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "open-msupply",
   "//": "Main version for the app, should be in semantic version format (any release candidate or test build should be separated by '-' i.e. 1.1.1-rc1 or 1.1.1-test",
-  "version": "2.0.02",
+  "version": "2.0.3",
```
Comment (roxy-dao) [inline package.json:4]:
```suggestion
  "version": "2.0.03",
```

Comment (CarlosNZ) [reply package.json]: Done --but why do we do this? It's not standard semantic versioning 🤷‍♂️
Screenshot 2024-06-05 at 2 30 17 PM

Comment (roxy-dao) [reply package.json]: actually.. was just following pattern but now you mentioned it it should be .1, .2, .3 not sure why the backend tests compare 0.1 > 0.01...

**Pull request merged**